x/claims/keeper/ibc_callbacks_test.go

```diff
@@ -559,11 +559,46 @@ func (suite *KeeperTestSuite) TestReceive() {
 				suite.Require().True(resAck.Success())
 
 				// check that the record is not deleted
-				suite.Require().True(suite.app.ClaimsKeeper.HasClaimsRecord(suite.ctx, receiver))
+				expCR := types.ClaimsRecord{
+					InitialClaimableAmount: sdk.NewInt(100),
+					ActionsCompleted:       []bool{false, false, false, true},
+				}
+
+				cr, found := suite.app.ClaimsKeeper.GetClaimsRecord(s.ctx, receiver)
+				suite.Require().True(found)
+				suite.Require().Equal(expCR, cr)
```

Comment:
Should this case also check that the sender's record is deleted?

Reply:
@jolube This is case 3, where the sender has no claims record, so it can't be deleted

Reply:
I was not hip to the nomenclature. Thanks